<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMQualiMatrixSettingsWidget</class>
 <widget class="QWidget" name="QMQualiMatrixSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>-1</number>
   </property>
   <property name="verticalSpacing">
    <number>20</number>
   </property>
   <item row="0" column="0" colspan="5">
    <widget class="QCheckBox" name="cbWriteTrainBottomToTop">
     <property name="text">
      <string>Schulungen von unten nach oben</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Höhe der vertikalen Leiste</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QSpinBox" name="sbQMVertWidth">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>400</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_7">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Farbe des Gitters</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QPushButton" name="pbGridColor">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>331</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_8">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Farbe der Selektion</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QPushButton" name="pbSelectionColor">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>331</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="5">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>413</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>345</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3" colspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>345</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_6">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Höhe der horizontalen Leiste</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QSpinBox" name="sbQMHorHeight">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>400</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbGridColor</sender>
   <signal>clicked()</signal>
   <receiver>QMQualiMatrixSettingsWidget</receiver>
   <slot>changeGridColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbSelectionColor</sender>
   <signal>clicked()</signal>
   <receiver>QMQualiMatrixSettingsWidget</receiver>
   <slot>changeSelectionColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbQMHorHeight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QMQualiMatrixSettingsWidget</receiver>
   <slot>emitSettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>541</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbQMVertWidth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QMQualiMatrixSettingsWidget</receiver>
   <slot>emitSettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>541</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbWriteTrainBottomToTop</sender>
   <signal>clicked()</signal>
   <receiver>QMQualiMatrixSettingsWidget</receiver>
   <slot>emitSettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeGridColor()</slot>
  <slot>changeSelectionColor()</slot>
  <slot>emitSettingsChanged()</slot>
 </slots>
</ui>
